Lib/test/test_math.py

```diff
@@ -1423,7 +1423,7 @@ def assertIsNaN(self, value):
 
 
 class IsCloseTests(unittest.TestCase):
-    isclose = math.isclose # sublcasses should override this
+    isclose = math.isclose # subclasses should override this
```

Comment:
For the comments on the same line as code, please leave 2 spaces before starting the comment.

If we choose to change the git blame for this line, then it makes sense to fix as much as we can :)

Reply:
Good point, done in https://github.com/python/cpython/pull/4803/commits/e2321c944031c8eeb87b7132e70e23c02900f20e